compose-lint-checks/src/test/java/slack/lint/compose/ParameterOrderDetectorTest.kt

```diff
@@ -120,7 +131,11 @@ class ParameterOrderDetectorTest : BaseComposeLintTest() {
           @@ -17 +17
           - fun MyComposable(text1: String, m2: Modifier = Modifier, modifier: Modifier = Modifier, trailing: () -> Unit) { }
           + fun MyComposable(text1: String, modifier: Modifier = Modifier, m2: Modifier = Modifier, trailing: () -> Unit) { }
-          """
+          Fix for src/test.kt line 20: Replace with (modifier: Modifier, text: String, lambda: Function0<Unit>):
+          @@ -20 +20
+          - inline fun <reified T> MyComposable(text: String = "123", modifier: Modifier = Modifier, lambda: () -> Unit) : T { }
+          + inline fun <reified T> MyComposable(modifier: Modifier, text: String, lambda: Function0<Unit>) : T { }
```

Comment:
Can we unpack lambdas?

Reply:
Let me check. I don't know how to get lambda when text is not available. I will check about sorting that differently.

Reply:
You should be able to just get the text range for that PSI element in the worst case, which would let you just get the `() ->` function syntax. My thinking is that this is the kind of thing that could be just a rearrangement of existing text, and you could do that replace in a stringy way

Reply:
Thank you for PSI element idea. I updated code to create correct error message and lint fix.